src/main/resources/templates/index.html

```diff
@@ -11,14 +11,14 @@
             width: 100%;
             padding: 16px 32px;
             text-alight: center;
-            font-size: 24px;
+            font-size: 25px;
         }
     </style>
 </head>
 <body>
-    <div class="container-fluid">
+    <section class="container-fluid">
         <div class="row">
-            <div class="col-md-12">
+            <div class="col-md-20">
```

Comment:
I changed the font size, I thought it appear better.

Reply:
That's a functionality change, and doesn't directly reduce technical debt.